### ISSUE TITLE
Picture should have an empty alt text

### ISSUE THREAD

**Issue by nemain00** [ALASdoo/alasdoo_com_v2#77]
Unnecessary image alt text on the picture on the "Contact us" page. Empty alt text should be put.
The issue is present on all three tabs.
